Source/SourceKittenFramework/LinuxCompatibility.swift

```diff
@@ -10,50 +10,30 @@ import Foundation
 
 extension Array {
     public func bridge() -> NSArray {
```

Comment:
Should we remove all these bridge functions?

Reply:
It may be so, but I think it would be better to have another PR.

Reply:
Agreed